package.json

```diff
@@ -7,10 +7,11 @@
     "codegen": "graph codegen",
     "build": "graph build",
     "deploy": "graph deploy fuseio/fuse-nft --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
-    "deploy-local": "graph deploy fuseio/fuse-nft --ipfs http://localhost:5001 --node http://127.0.0.1:8020"
+    "deploy-local": "graph deploy fuseio/fuse-nft --ipfs http://localhost:5001 --node http://127.0.0.1:8020",
+    "deploy-hirenodes": "graph deploy fuseio/fuse-nft --ipfs http://64.226.100.49:5001 --node http://64.226.100.49:8020"
```

Comment:
Can you please ask hirenodes for a URL?

Reply:
@leonprou yeah we have the URL https://graph-fuse-mainnet.hirenodes.io, I'll update the PR